akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/scaladsl/ScalaTestWithActorTestKit.scala

```diff
@@ -7,26 +7,21 @@ package akka.actor.testkit.typed.scaladsl
 import akka.actor.testkit.typed.TestKitSettings
 import com.typesafe.config.Config
 import com.typesafe.config.ConfigFactory
-import org.scalatest.BeforeAndAfterAll
-import org.scalatest.Matchers
-import org.scalatest.WordSpecLike
+import org.scalatest._
 import org.scalatest.concurrent.Eventually
 import org.scalatest.concurrent.ScalaFutures
 import org.scalatest.time.Span
 
 /**
  * A ScalaTest base class for the [[ActorTestKit]], making it possible to have ScalaTest manage the lifecycle of the testkit.
- * The testkit will be automatically shut down when the test completes or fails.
+ * The testkit will be automatically shut down when the test completes or fails using ScalaTest's BeforeAndAfterAll trait. If
+ * a spec overrides afterAll it must call super.afterAll.
  *
  * Note that ScalaTest is not provided as a transitive dependency of the testkit module but must be added explicitly
  * to your project to use this.
- *
- * This is a `WordSpec`, other ScalaTest styles can be implemented in a similar abstract class like this one.
- * For example `with FlatSpecLike with BeforeAndAfterAll` and implementing the `afterAll` method by calling
- * `testKit.shutdownTestKit()`.
  */
-abstract class ActorTestKitWordSpec(testKit: ActorTestKit) extends ActorTestKitBase(testKit)
-  with WordSpecLike with Matchers with BeforeAndAfterAll with ScalaFutures with Eventually {
+abstract class ScalaTestWithActorTestKit(testKit: ActorTestKit) extends ActorTestKitBase(testKit)
+  with TestSuite with Matchers with BeforeAndAfterAll with ScalaFutures with Eventually {
```

Comment:
nice change, since some people may want `FlatSpec` etc 👍 

Name I'd consider using the *Support naming we have in some places in akka http; like "ScalaTestActorTestKitSupport"? If you want to keep though then I can live with this one

Reply:
I don't think "support" adds much information. Happy with current.